examples/src/main/java/com/cloudera/nav/sdk/examples/schema/FireCircleDataset.java

```diff
@@ -61,5 +62,9 @@ public void setDataContainer(HdfsEntity hdfsDir) {
     Preconditions.checkArgument(hdfsDir.getEntityType() ==
         EntityType.DIRECTORY);
     this.dataContainer = hdfsDir;
+    this.dataContainer.setIsEndPoint(true);
   }
+
+  @Override
+  public void validateEntity() {}
```

Comment:
If you're having to create empty stubs all around your code like this, you're probably not doing the right thing